src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -88,7 +88,7 @@ private static DetailNode parseJavadocAsDetailNode(String javadocComment) {
      * @param parseErrorMessage ParseErrorMessage
      * @return error message
      */
-    private static String getParseErrorMessage(ParseErrorMessage parseErrorMessage) {
+    public static String getParseErrorMessage(ParseErrorMessage parseErrorMessage) {
```

Comment:
@ps-sp , please always try to use public methods to test smth.
you can use reflection only and only when you have no way to test it by public method, such usage of reflection should be commented in code.